src/language/mongoDBService.ts

```diff
@@ -245,9 +267,16 @@ export default class MongoDBService {
           )
         );
 
-        worker?.on(
-          'message',
-          ({ error, data }: { data?: ShellEvaluateResult; error?: any }) => {
+        worker?.on('message', ({ name, payload }) => {
+          if (name === ServerCommands.SHOW_CONSOLE_OUTPUT) {
+            void this._connection.sendNotification(name, payload);
+          }
+
+          if (name === ServerCommands.EXECUTE_CODE_FROM_PLAYGROUND) {
```

Comment:
Not quite related to these changes, currently we use this `EXECUTE_CODE_FROM_PLAYGROUND` command enum both for the request to the language server worker, and for the response from the worker. 
How do you feel about having two separate commands here? That way it's a bit easier to follow and search in the code. Maybe `EXECUTE_CODE_FROM_PLAYGROUND_REQUEST` and `EXECUTE_CODE_FROM_PLAYGROUND_RESPONSE`? 
Maybe we can have a separate enum called `WorkerCommands` to help distinguish the two?

Reply:
Good call out, I'll add a new `WorkerCommands` enum.

Reply:
Have opted to introduce a new server command `CODE_EXECUTE_RESULT` for sending the result from worker. As we will still need 2 different commands with a new `WorkerCommands` enum. I didn't change `EXECUTE_CODE_FROM_PLAYGROUND` to minimize the changes.